misc/helm-chart/create-pr.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+VERSION=$1
+
+# Update version in file
+echo "Update Chart.yaml with Trivy $VERSION"
+sed -i "s/version: [0-9]\+\.[0-9]\+\.[0-9]\+/version: $VERSION/" ./helm/trivy/Chart.yaml
+sed -i "s/appVersion: [0-9]\+\.[0-9]\+\.[0-9]\+/appVersion: $VERSION/" ./helm/trivy/Chart.yaml
+
+echo "Create PR for update Trivy $VERSION in the Helm Chart"
+
+# Create a new branch
+NEW_BRANCH="ci/bump-trivy-to-$VERSION"
```

Comment:
I think we need to say about `helm` in branch name, eg:
```suggestion
NEW_BRANCH="ci/bump-helm-trivy-to-$VERSION"
```

Reply:
done [5023312](https://github.com/aquasecurity/trivy/pull/7526/commits/5023312fdf5e3fa97ac5bd17d2760bd8d77b57e9)